### DISCUSSION TITLE
Does This FHIR convert not support MAC.

### DISCUSSION THREAD

**Discussion by SandeepRDiddi** [microsoft/FHIR-Converter#284] [General]
I installed FHIR extension on VS Code on my Mac book but when i try to conver to FHIR bundle i get below message 

Conversion Error - FHIR Converter is not supported in this operating system now.

Any leads on how can we use this converter on Mac book

**Comment by ginalee-dotcom** [comment]
Currently, FHIR converter and the VS Code extension for FHIR converter are supported only on Windows. We are planning to make them available on Mac and other operating systems in the near future. We will keep you updated once this becomes available!

**Comment by granadacoder** [comment]
You can pull the code and then use dotnet.exe to "publish" a run-time that ~should work on linux/mac-something-like-linux.


(Terminal Window (on Mac))

// change directory to where the .sln file is
    
     $ dotnet publish

//wait a few minutes

// now there is a "publish" directory .. 

    $ cd ./src/Microsoft.Health.Fhir.Liquid.Converter.Tool/bin/Debug/netcoreapp3.1/publish

now the below command runs (note, the no ".exe" on it of course!  its a mac)
 
     $ ./Microsoft.Health.Fhir.Liquid.Converter.Tool --version

and the output:

    Microsoft.Health.Fhir.Liquid.Converter.Tool 1.0.0
